src/actions.rs

```diff
@@ -47,13 +47,39 @@ lazy_static! {
     };
 }
 
+// Group all the pending FCP for these teams
+pub const IN_PRE_FCP_COMPILER_TEAM: &str = "in_pre_fcp_compiler_team";
+pub const IN_PRE_FCP_RUST: &str = "in_pre_fcp_rust";
+pub const IN_PRE_FCP_FORGE: &str = "in_pre_fcp_forge";
+const PENDINGFCP: [&str; 3] = [IN_PRE_FCP_COMPILER_TEAM, IN_PRE_FCP_RUST, IN_PRE_FCP_FORGE];
+
+// Group all the FCP for these teams
+pub const IN_FCP_COMPILER_TEAM: &str = "in_fcp_compiler_team";
+pub const IN_FCP_RUST: &str = "in_fcp_rust";
+pub const IN_FCP_FORGE: &str = "in_fcp_forge";
+const THINGSINFCP: [&str; 3] = [IN_FCP_COMPILER_TEAM, IN_FCP_RUST, IN_FCP_FORGE];
+
+// Group all the Finalized FCP for these teams
+pub const FCP_FINISHED_COMPILER_TEAM: &str = "fcp_finished_compiler_team";
+pub const FCP_FINISHED_RUST: &str = "fcp_finished_rust";
+pub const FCP_FINISHED_FORGE: &str = "fcp_finished_forge";
+const FINALIZEDFCP: [&str; 3] = [
+    FCP_FINISHED_COMPILER_TEAM,
+    FCP_FINISHED_RUST,
+    FCP_FINISHED_FORGE,
+];
```

Comment:
All the github queries names for FCPs are now listed here, using constants to avoid typos .